src/go/rpk/pkg/cli/acl/acl.go

```diff
@@ -83,6 +83,21 @@ in the --allow-principal and --deny-principal flags. This command will add the
 "User:" prefix for you if it is missing. The wildcard '*' matches any user.
 Creating an ACL with user '*' grants or denies the permission for all users.
 
+ROLES
+
+Alternatively, an ACL may be bound to a role. To create a role-bound ACL,
+follow the same procedure used to create a user ACL, but use the --allow-role
```

Comment:
[question] IIUC, `--allow-principal RedpandaRole:role"` is no longer the preferred way? But it's still an option but not documented?

Reply:
Sounds good to me. As long as we document this as an alternative (it might be confusing) I'm fine :+1:

Reply:
>as long as we document this as an alternative

Ah, so you're saying you prefer the old usage to be documented? I was thinking omission might be less confusing, but I could go either way.

Reply:
Oh I'm so sorry! I meant `as long as we _dont_ document this as an alternative`

Reply:
No worries! that's what I thought but didn't want to presume 🙂